packages/env/lib/init-config.js

```diff
@@ -80,6 +119,14 @@ module.exports = async function initConfig( {
 			yaml.dump( dockerComposeConfig )
 		);
 
+		// Add multisite .htaccess if multisite is enabled
+		if ( config.env.multisite ) {
+			await writeMultisiteHtaccess( '/var/www/html' );
```

Comment:
This is a drive-by remark (I'm on my phone and I'll have a better look in the morning), but isn't this saving to the _host's_ own filesystem? Isn't there a path.join missing, or a docker-compose mount missing?

Reply:
The path join is added in line 48. It is creating the file on the Dockerfile.